playground/load_your_code.md

```diff
@@ -0,0 +1,594 @@
+         # Dataset id. Will be used as a topic name.
+         id: dataset
+         # Name of dataset specified in "datasets" section.
+         source_dataset: "CountWords"
+```
+
+For metadata reference see fields in "Tag" class [here](infrastructure/models.py).
+
+##### Default examples
+
+Each SDK must have a single default example.
+If there is none, the user will see the error in the app and face a blank editor.
+If there are more than one, it is not defined which one will be selected.
+
+##### Kafka emulator
+
+Examples which require Kafka server emulator, need to include the `emulators` tag
+and provide `dataset` in the example's tag. You can refer to an example
+[here](/examples/java/src/main/java/org/apache/beam/examples/KafkaWordCountJson.java).
+
+1. Add your dataset in either JSON or Avro format into the `playground/backend/datasets` path.
+
+2. Add the following elements to the example's metadata tag:
+    ```YAML
+    emulators:
+      - type: kafka
+        topic:
+          id: dataset
+          source_dataset: <dataset_name>
+    datasets:
+      <dataset_name>:
+        location: local
+        format: json # or 'avro'
+    ```
+    replace `<dataset_name>` with the name of your dataset file without the file name extension.
+
+3. Use the exact string `"kafka_server:9092"` as a server name in your code snippet.
+   This string will be replaced by the actual host name and port automatically
+   before the compilation step by Playground.
+
+>**Kafka emulator limitations:**
+> - Playground Kafka emulator currently supports only Beam Java SDK
+> - exact string `"kafka_server:9092"` should be present in the code snippet;
+    any other variation like `"kafa_server" + ":9092"` will not work
+
+#### 3. Make a PR
+
+Make a PR with code snippet into [the Apache Beam repository](https://github.com/apache/beam)
+following the [Contribution guide](https://beam.apache.org/contribute/).
+Verify that all pre-commit tests are passing.
+
+Playground CI will verify and deploy the example to Playground Example Catalog when the PR is merged.
+
+#### 4. Save the snippet ID
+
+The snippet will be assigned an ID.
+You can find it in the address bar of the browser when you select it in the dropdown.
+
+For example, in this URL:
+
+```
+https://play.beam.apache.org/?path=SDK_JAVA_MinimalWordCount&sdk=java
+```
+
+the ID is: `SDK_JAVA_MinimalWordCount`.
+
+You will need the snippet ID to embed the Playground with snippet into a website page.
+
+### Source 2. How to add an unlisted example to Playground
+
+Not all examples must be visible in the example dropdown.
+Some examples are best in the context of Apache Beam documentation.
+To embed them into the documentation use unlisted examples.
+They work and are checked and cached the same way as public examples.
+
+Proceed the same way as with [Source 1. Playground Examples Catalog](#source-1-how-to-add-an-example-to-playground-examples-catalog) except:
+1. Use the directory `/learning/beamdoc`
```

Comment:
Replace "use" to something more meaningful, e.g. add an example file to the directory ./beam/learning/beamdoc/<sdk>.
It'd be better to have a dedicated subfolder for each SDK